agent/engine/task_manager.go

```diff
@@ -345,8 +345,8 @@ func (mtask *managedTask) waitEvent(stopWaiting <-chan struct{}) bool {
 		mtask.handleDesiredStatusChange(acsTransition.desiredStatus, acsTransition.seqnum)
 		return false
 	case dockerChange := <-mtask.dockerMessages:
-		seelog.Infof("Managed task [%s]: got container [%s] event: [%s]",
-			mtask.Arn, dockerChange.container.Name, dockerChange.event.Status.String())
+		seelog.Infof("Managed task [%s]: got container [%s (Docker ID: %s)] event: [%s]",
+			mtask.Arn, dockerChange.container.Name, dockerChange.container.RuntimeID, dockerChange.event.Status.String())
```

Comment:
RuntimeID is not read-only (can be set by SetRuntimeID) so read should be protected by lock with GetRuntimeID(), same for other places

Reply:
Thanks for the suggestion. Changed it.